deploy/kubeconfiggenerator.py

```diff
@@ -488,62 +488,56 @@ def flatten(yaml_contents, flattened, types_dict, prefix=''):
             types_dict[key] = {'type': 'integer'}
         elif isinstance(value, float):
             flattened[prefix + key] = value
-            types_dict[key] = {'type': 'float'}
+            types_dict[key] = {'type': 'number'}
         if isinstance(value, dict):
             inner_prop_dict = {}
             prop_dict = {'properties': inner_prop_dict}
             prop_dict['type'] = 'object'
-            prop_dict['additionalProperties'] = True
```

Comment:
Why do we have to remove this line?

Setting additionalProperties to 'true' enables specifying properties in annotations on a resource that have not been explicitly defined in the resource spec. This was a requirement for one of our users and the code update came from them.